package.json

```diff
@@ -15,16 +15,10 @@
     "pretest": "npm run lint",
     "test": "NODE_ENV=test nyc mocha"
   },
-  "author": "@diffcunha",
   "repository": {
     "type": "git",
-    "url": "git://github.com/diffcunha/jsonmltoreact.git"
+    "url": "https://github.com/CondeNast/jsonmltoreact"
   },
-  "license": "MIT",
```

Comment:
honestly, I don't know the rules of the internet that govern this type of thing.  I was just mirroring the [package.json](https://github.com/CondeNast/autopilot-models/blob/master/package.json) in autopilot-models, but not sure what the conventions are when you fork things.  Since this was originally created with MIT license, I wonder if that means we have to do anything special?  I suppose I should read the license...